docs/modules/nifi/pages/getting_started/installation.adoc

```diff
@@ -3,23 +3,26 @@
 On this page you will install the Stackable Operator for Apache NiFi and operators for its dependencies - ZooKeeper -
 as well as the commons, secret and listener operator which are required by all Stackable Operators.
 
-== Stackable Operators
+== Stackable operators
 
 There are two ways to install Stackable Operators:
 
-. Using xref:management:stackablectl:index.adoc[stackablectl]
-. Using Helm
-
-=== stackablectl
+* Using xref:management:stackablectl:index.adoc[stackablectl] (recommended)
```

Comment:
I mainly take issue with recommending a CLI tool for operating in production.

IMO, Helm (or something that can work in a gitops workflow) would be the recommended way in production.

I'm sure it is subjective, so I won't argue it here and I truly don't mind if we recommend the CLI for any case, but I think it should then be explained as to why.

Or, we could recommend either option depending on the style of the end user and their policies. Eg: We recommend 
- `stackablectl` for human-operated installation
- `helm` for gitops management of infrastructure

Reply:
After writing the _note_ in the above comment, I realised my problem between the two styles is about lineage.

If we could add in stuff to labels/annotation to explain how they got there (stackablectl version, CLI parameters that influence the deployed manifests).

Eg: Given an installation in a k8s cluster, how can I perfectly replicate that from `stackablectl`:
- Which CLI parameters were passed in.
- Which version of stackablectl (AFAIK, we don't store the commit hash in the binary, but if we did, that could go into labels)

Reply:
> question: recommended for what? All clusters, or just getting started?

I would say recommended for "trying out demos and stacks". The CLI is currently not in a place to be used as a production-ready management / deployment tool. It might be capable to fill that role in the future, but not right now.

Reply:
> If we could add in stuff to labels/annotation to explain how they got there (stackablectl version, CLI parameters that influence the deployed manifests).

I agree that we need to do a lot better indicating what was installed by us and how. This is an ongoing (but rather tricky) effort.

Reply:
Okay, so what is the recommended installation procedure for production then? I think we should probably document that somewhere.

Reply:
I would assume by elimination is has to be Helm (and I guess for OpenShift, it is OLM?)

Reply:
Lets just remove the `(recommended)` text for now...

```suggestion
* Using xref:management:stackablectl:index.adoc[stackablectl]
```